### DISCUSSION TITLE
Adding winget as an installation method

### DISCUSSION THREAD

**Discussion by johndoe432** [FreeTubeApp/FreeTube#2057] [General]
Project's readme already suggests downloading FreeTube from package repos (AUR or Chocolatey). 

Winget is also a pretty popular option. It simply allows user to download an installer from this repo and automatically installs it and also allows simply updating FreeTube by running one command.

Winget repo already contains a FreeTube page, which is maintained by a moderator, so there are no risks of catching something malicious.

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
i'll add it to the readme!

edit: opened up PR https://github.com/FreeTubeApp/FreeTube/pull/2143
